### DISCUSSION TITLE
Looking for a piece of code

### DISCUSSION THREAD

**Discussion by deno555** [eritislami/evobot#804] [Q&A]
Im trying to find the code that prevents the bot from responding to other bots. I want to use the bot for playing music on a private server.

**Comment by 0zul** [comment]
```js
if(message.author.bot) {
return;
}
```

That should do.

**Reply by deno555** [reply]
is it in the code or should i put it in?
and where is it / put it

**Comment by wolf-yuan-6115** [comment]
Open bot main file and find `client.on("message", ...)`, add
```js
if (message.author.bot) return;
```
In first line of that callback.
